### PR TITLE
Update license

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Use latest version of the license. See https://creativecommons.org/licenses/by/4.0/.

### CODE REVIEW

**Review by samthor — APPROVED** [review]

**Comment by robdodson** [comment]
Hey @tomayac are you cool with us merging this or did you still want to update the LICENSE file?

File: site/_data/i18n/footer.yml [hunk 1]
```diff
@@ -50,4 +50,4 @@ terms:
   en: Terms
 
 content_license:
-  en: Content available under the CC-By 3.0 license
\ No newline at end of file
+  en: Content available under the CC BY 4.0 license
```
Comment (tomayac) [inline site/_data/i18n/footer.yml:53]:
```suggestion
  en: Content available under the CC-BY-SA-4.0 license
```

Comment (tomayac) [inline site/_data/i18n/footer.yml:53]:
```suggestion
  en: Content available under the CC-BY-SA-4.0 license
```

**Comment by tomayac** [comment]
> Hey @tomayac are you cool with us merging this or did you still want to update the LICENSE file?

@robdodson This is good to be merged now. Thanks!

**Pull request merged**